framework/include/utils/MooseUtils.h

```diff
@@ -316,6 +316,12 @@ indentMessage(const std::string & prefix, std::string & message, const char * co
  */
 std::string & removeColor(std::string & msg);
 
+std::list<std::string> listDir(const std::string path, bool files_only = false);
```

Comment:
Please add documentation to these new and changed utilities.

Reply:
Will do.